ci/tasks/run-int.yml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source: {repository: everlag/fog-google}
```

Comment:
How would you like to proceed?

I'm happy to keep `everlag/fog-google` available but it'd be better if maintainers had direct access to update the image in the future. To build and upload the image to an account should be a two command process once docker is installed and authenticated.